lib/components/admin/field-trip-windows.js

```diff
@@ -0,0 +1,215 @@
+                      fontSize: 'revert',
+                      fontWeight: 400,
+                      marginRight: '15px'
+                    }}
+                    placeholder='Search'
+                    className='pull-right'
+                    value={filter.search}
+                    onChange={this._onSearchChange} />
+                </h4>
+                {TABS.map(tab => {
+                  const active = tab.id === filter.tab
+                  const style = {
+                    borderRadius: 5,
+                    backgroundColor: active ? 'navy' : undefined,
+                    color: active ? 'white' : undefined,
+                    padding: '2px 3px'
+                  }
+                  const requestCount = fieldTrip.requests.data.filter(tab.filter).length
+                  return (
+                    <button
+                      key={tab.id}
+                      name={tab.id}
+                      style={style}
+                      className='clear-button-formatting'
+                      onClick={this._onTabChange}
+                    >
+                      {tab.label} <Badge>{requestCount}</Badge>
+                    </button>
+                  )
+                })}
+              </>
+            }
+            onClickClose={toggleFieldTrips}
+            style={{width: '450px'}}
+          >
+            {visibleRequests.length > 0
+              ? visibleRequests.map((request, i) => (
+                <FieldTripRequestRecord
+                  key={request.id}
+                  active={activeId === request.id}
+                  onClick={this._onClickFieldTrip}
+                  request={request} />
+              ))
+              : <div>No field trips found.</div>
+            }
+          </DraggableWindow>
+          : null
+        }
+        <FieldTripDetails />
+      </>
+    )
+  }
+}
+
+class FieldTripRequestRecord extends Component {
+  _onClick = () => {
+    const {onClick, request} = this.props
+    onClick(request)
+  }
+
+  _getStatusIcon = (status) => status
+    ? <Icon type='check' className='text-success' />
+    : <Icon type='exclamation-circle' className='text-warning' />
+
+  render () {
+    const {active, request} = this.props
+    const style = {
+      backgroundColor: active ? 'lightgrey' : undefined,
+      borderBottom: '1px solid grey'
+    }
+    const {
+      endLocation,
+      id,
+      inboundTripStatus,
+      outboundTripStatus,
+      schoolName,
+      startLocation,
+      teacherName,
+      timeStamp
+    } = request
+    return (
+      <li
+        className='list-unstyled'
+        style={style}
+      >
+        <button
+          className='clear-button-formatting'
+          style={{width: '-webkit-fill-available', display: 'inline-block'}}
+          name={id}
+          onClick={this._onClick}
+        >
+          <div style={{fontWeight: 600, display: 'inline-block', width: '50%'}}>{schoolName} Trip (#{id})</div>
```

Comment:
Line too long

Reply:
fixed in 88238d3fc548beeee1a929a280aa4639334af21c